PowerPointLabs/PowerPointLabs/NarrationsLab/Views/NarrationsLabMainSettingsPage.xaml.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using PowerPointLabs.ActionFramework.Common.Log;
+using PowerPointLabs.NarrationsLab.Data;
+using PowerPointLabs.NarrationsLab.Storage;
+using PowerPointLabs.NarrationsLab.ViewModel;
+using PowerPointLabs.TextCollection;
+
+namespace PowerPointLabs.NarrationsLab.Views
+{
+    /// <summary>
+    /// Interaction logic for NarrationsLabMainSettingsPage.xaml
+    /// </summary>
+    public partial class NarrationsLabMainSettingsPage : Page
+    {
+        public delegate void DialogConfirmedDelegate(string voiceName, AzureVoice azureVoiceName, bool isAzureVoiceSelected, bool preview);
```

Comment:
Should bool preview have a different name (eg, isPreviewing)?

Reply:
Edited!